client/homebrew/pages/vaultPage/vaultPage.jsx

```diff
@@ -0,0 +1,473 @@
+require('./vaultPage.less');
+
+const React = require('react');
+const { useState, useEffect, useRef } = React;
+const cx = require('classnames');
+
+const Nav = require('naturalcrit/nav/nav.jsx');
+const Navbar = require('../../navbar/navbar.jsx');
+const RecentNavItem = require('../../navbar/recent.navitem.jsx').both;
+const Account = require('../../navbar/account.navitem.jsx');
+const NewBrew = require('../../navbar/newbrew.navitem.jsx');
+const HelpNavItem = require('../../navbar/help.navitem.jsx');
+const BrewItem = require('../basePages/listPage/brewItem/brewItem.jsx');
+const SplitPane = require('../../../../shared/naturalcrit/splitPane/splitPane.jsx');
+
+const request = require('../../utils/request-middleware.js');
+
+const VaultPage = (props) => {
+    const [title, setTitle] = useState(props.query.title || '');
```

Comment:
Does our linter handle jsx? because this code has been written, rewritten, and past through many places, i can take a look at my prettier config to see if i can change those to tabs.